Tools/scripts/ok.txt

```diff
@@ -0,0 +1,7 @@
+  ______    __  ___ 
```

Comment:
Why do we need this?

Reply:
Why not? It is like a little easter egg in our code!